packages/domain-picker/test/suggestion-item.tsx

```diff
@@ -1,14 +1,25 @@
 /**
  * @jest-environment jsdom
  */
+// https://jestjs.io/docs/en/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom
+import '../matchMedia.mock';
 
 /**
  * External dependencies
  */
 import * as React from 'react';
 // eslint-disable-next-line import/no-extraneous-dependencies
-import { render, fireEvent } from '@testing-library/react';
+import { render, fireEvent, screen } from '@testing-library/react';
+import SuggestionItem from '../src/domain-picker/suggestion-item';
```

Comment:
Similarly to https://github.com/Automattic/wp-calypso/pull/45490/files#r486327394 — in other files we have external and internal deps separated in their own sections. I think there's a linter rule for that too.